images-list

```diff
@@ -12,6 +12,8 @@ banzaicloud/logging-operator rancher/banzaicloud-logging-operator 3.7.3
 banzaicloud/logging-operator rancher/banzaicloud-logging-operator 3.8.0
 bats/bats rancher/bats-bats v1.1.0
 coredns/coredns rancher/coredns-coredns 1.7.0
+coredns/coredns rancher/coredns-coredns 1.7.1
+coredns/coredns rancher/coredns-coredns 1.8.0
```

Comment:
I have a question here - why do we maintain multiple versions of the same image?

Reply:
Because different versions of k3s/rke2/rke use different versions of the image, and all versions need to be present in our mirror.